addons/web/static/src/views/relational_model.js

```diff
@@ -1813,6 +1823,22 @@ export class DynamicRecordList extends DynamicList {
     // Getters
     // -------------------------------------------------------------------------
 
+    get commonRecordParams() {
+        return {
+            resModel: this.resModel,
+            fields: this.fields,
+            activeFields: this.activeFields,
+            onRecordWillSwitchMode: this.onRecordWillSwitchMode,
+            defaultContext: this.defaultContext,
+            rawContext: {
+                parent: this.rawContext,
+                make: () => this.context,
+            },
+            onWillSaveRecord: this.onWillSaveRecord,
+            onSavedRecord: this.onSavedRecord,
+        };
```

Comment:
maybe sort this somehow (at least regroup callbacks)

Reply:
I don't understand ?